src/alg_utils.jl

```diff
@@ -1,5 +1,7 @@
 isautodifferentiable(alg::OrdinaryDiffEqAlgorithm) = true
 
+DiffEqBase.isdiscrete(alg::FunctionMap) = true
```

Comment:
This should probably be used in place of things like https://github.com/JuliaDiffEq/DiffEqBase.jl/blob/master/src/solutions/solution_interface.jl#L61